integration/ctr/oci_hooks_tests.bats

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2022 Apple Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE="${IMAGE:-docker.io/library/busybox:latest}"
+CONTAINER_NAME="${CONTAINER_NAME:-kataocihookstest}"
+CTR_RUNTIME="${CTR_RUNTIME:-io.containerd.kata.v2}"
```

Comment:
Do you need to test any other runtime than `io.containerd.kata.v2`? If not, I would hard-code that value, because otherwise you might end up on testing a different runtime by mistake (suppose the environment have `CTR_RUNTIME` exported with an unexpected value).

Reply:
I picked this up from another file as a "pattern" I assumed, so happy to switch back to hardcoding it to `io.containerd.kata.v2`!